client/src/components/Feed/StyledPostTabCard.js

```diff
@@ -0,0 +1,102 @@
+import styled from "styled-components";
+import { theme } from "constants/theme";
+
+const { colors, typography } = theme;
+const { xxsmall, xsmall, medium } = typography.size;
+
+export const StyledPostTabCard = styled.div`
+  display: flex;
+  background-color: white;
+  width: 100%;
+`;
+
+export const StyledCardHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+`;
+
+export const StyledCardBody = styled.div`
+  border-top: unset;
+  padding: 0 1.8rem;
+  margin-top: 0.7rem;
+  font-family: "Work Sans";
+  white-space: pre-line;
+  font-weight: 400;
+  width: 100%;
+`;
+
+export const StyledUpdateAt = styled.p`
+  font-size: ${xxsmall};
+  line-height: 1.4rem;
+  color: #939393;
```

Comment:
minor: it's better to use the constant color instead of hard-coded values, like this one equals the `DARK_GRAY`